### ISSUE TITLE
Blind Killian guard(s)

### ISSUE THREAD

**Issue by wipe2238** [rotators/Fo1in2#25]
If you're helping to deal with Kenji, there's a chance one of guards (most likely the one near window) misses and hits player ("_Oops!_"). Once Kenji is done everyone in store attacks player's party even if there's no reason to be so rude (player hides weapon after dealing killing blow, none of party members missed during fight).

**Comment by ghost2238** [comment]
@Lexx2k - is this something that can be fixed with scripts or does it need engine work possibly?

**Comment by Lexx2k** [comment]
I can't fix this. It's simply how Fallout combat works if critters aren't in the same team.

**Comment by ghost2238** [comment]
Alright. Would it be possible to put the player on the same team as the ally NPCs temporarily during such scenarios? I guess this also includes attacking regulators in boneyard.

**Comment by Lexx2k** [comment]
Well, yes, but I won't do it. You can't prevent the bug anyway, it'll just happen in other combat encounters then. This bug even exists in Fo1, where damaged brahmin in caravan encounters would fuck up your life.

Either someone fixes this in engine ("Ooops"-damage won't make critter hostile), or players will simply have to live with it.

**Comment by ghost2238** [comment]
> 
> You can't prevent the bug anyway, it'll just happen in other combat encounters then. This bug even exists in Fo1, where damaged brahmin in caravan encounters would fuck up your life.

Yeah I remember this from playing back in the day. A lot of save game reloading due to mad brahmins.

> 
> Either someone fixes this in engine ("Ooops"-damage won't make critter hostile), or players will simply have to live with it.

Let's leave it for the time being then. Maybe someone else (not very likely) will one day fix it.

**Comment by FakelsHub** [comment]
You're serious?! This is not a bug.

**Comment by Lexx2k** [comment]
I dunno. If a critter gets accidentally hit, it sucks hard when they turn hostile. Especially if it's completely out of the players hands.

**Comment by ghost2238** [comment]
> 
> 
> You're serious?! This is not a bug.

I would be very surprised if the original developers intended this to happen, but sure, let's consider it a feature even if it makes no sense.

**Comment by wipe2238** [comment]
> You're serious?! This is not a bug.

I feel something got lost in translation...

_Oops_ <- not a bug
NPC attacking player **after NPC's _oops_** <- bug

**Comment by Lexx2k** [comment]
Just got reminded of this shit. Stupid guard hit me and then suddenly everyone ganged up on me. This is odd as hell. It was *his* mistake, not mine.
